tests/foreman/destructive/test_clone.py

```diff
@@ -44,7 +44,7 @@ def test_positive_clone_backup(
 
     :parametrized: yes
 
-    :BZ: 2142514, 2013776
+    :Verifies: SAT-10789, SAT-15437, SAT-13950
```

Comment:
we removed 2 bz's and how we've 3 jira issues for it :)

Reply:
Fixed now.